xml/art_amd-sev.xml

```diff
@@ -340,6 +340,43 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec-amd-sev-kubevirt">
+  <title>SEV with KubeVirt</title>
+  <para>
+   KubeVirt supports running of SEV guests starting from the version <literal>0.49.0</literal>. The
+   functionality can be activated by enabling the <literal>WorkloadEncryptionSEV</literal> feature
+   gate:
+  </para>
+  <screen>&prompt.user;kubectl edit kubevirt kubevirt -n kubevirt
+   ...
+   spec:
+     configuration:
+       developerConfiguration:
+         featureGates:
+         - WorkloadEncryptionSEV
+   ...
+  </screen>
+  <para>
+   In order to run an SEV-encrypted guest the VMI spec must include the entry
+   <tag class="element">sev: {}</tag> under <tag class="element">launchSecurity</tag> domain
```

Comment:
Why using &lt;tag&gt; here? Is the config file XML so that it requires sharp brackets <>? If you only want to highlight a keywork, use &lt;literal/&gt;

Reply:
I just wasnt sure what to use here. Switched to using `<literal>` as suggested.

Reply:
thanks!